packages/keplr-stores/src/price/index.ts

```diff
@@ -350,7 +351,8 @@ export class CoinGeckoPriceStore extends ObservableQuery<CoinGeckoSimplePrice> {
     coin: CoinPretty,
     vsCurrrency?: string
   ): PricePretty | undefined {
-    if (!coin.currency.coinGeckoId) {
+    const currency = coin.currency as AppCurrency;
+    if (!currency.coinGeckoId && !currency.priceCoinId) {
```

Comment:
What is this new PriceCoinId concept? (Haven't gotten to assetlist change, just didn't see it in github PR description)

Reply:
PriceCoinId is an old property of our `source-chain-config.ts` used to calculate prices based on pools instead of Coingecko. Assetlist will replace this with the `osmosis-price` located within the keyword array. However, it's currently not ready as it does not support CL pools yet. Once it's ready, we'll have to make some changes to use the keyword array and remove the `prices.ts`.